### PR TITLE
No matplotlib on Windows+Python 3.5

### PR DESCRIPTION
**Pull request by ocefpaf**
**DO NOT MERGE!**

Just testing. Nothing to see here. Move along!

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,18 +21,18 @@ requirements:
...
-        - six
         - rtree
-        - descartes
-        - matplotlib
-        - psycopg2  # [not win]
-        - sqlalchemy
-        - geopy ==1.10.0
+        - geopy 1.10.0
         - pysal
+        - matplotlib  # [not win and py35]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @jorisvandenbossche it was `matplotlib`! I am trying to fix it now so we can ship a fully functioning `geopandas`.

**Comment by ocefpaf** [comment]
The Linux failure is new :smile: 

```shell
===== testing package: geopandas-0.2-py27_0 =====
import: 'geopandas'
Traceback (most recent call last):
  File "/feedstock_root/build_artefacts/test-tmp_dir/run_test.py", line 26, in <module>
    import geopandas
  File "/opt/conda/envs/_test/lib/python2.7/site-packages/geopandas/__init__.py", line 4, in <module>
    from geopandas.io.file import read_file
  File "/opt/conda/envs/_test/lib/python2.7/site-packages/geopandas/io/file.py", line 3, in <module>
    import fiona
  File "/opt/conda/envs/_test/lib/python2.7/site-packages/fiona/__init__.py", line 69, in <module>
    from fiona.collection import Collection, BytesCollection, vsi_path
  File "/opt/conda/envs/_test/lib/python2.7/site-packages/fiona/collection.py", line 7, in <module>
    from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
ImportError: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by /opt/conda/envs/_test/lib/python2.7/site-packages/fiona/../../../libgdal.so.20)
TESTS FAILED: geopandas-0.2-py27_0

./ci_support/run_docker_build.sh returned exit code 1
```

**Comment by jorisvandenbossche** [comment]
But that's probably an issue with the new fiona/gdal version? (previously it was still using gdal 1)

**Comment by ocefpaf** [comment]
> But that's probably an issue with the new fiona/gdal version? (previously it was still using gdal 1)

Yep. I am investigating this here. That is puzzling b/c `GLIBCXX_3.4.15` comes from the docker image and that is always present.

**Comment by ocefpaf** [comment]
Found the problem. [`fiona`](https://circleci.com/gh/conda-forge/fiona-feedstock/40) (and probably `gdal` too) is pulling `conda-forge`'s `numpy` that is compiled against `openblas` and, b/c of that, uses `libgcc`: 

```shell
numpy-1.10.4               |py27_blas_openblas_201         6.8 MB  conda-forge
```

but geopandas is pulling the default channel `numpy` that is not using `openblas` (probably `mkl`).

```shell
numpy-1.11.0               |           py27_1         6.2 MB  defaults
```

That is not the first time our `numpy` breaks things... I will see what I can do. Meanwhile I will add a (horrible) workaround for this.

**Comment by jorisvandenbossche** [comment]
@ocefpaf In any case (this goes a bit over my head), thanks a lot for all your work on this!

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipe/meta.yaml]: @jorisvandenbossche I did not merge this b/c `geopandas` without `matplotlib` is no fun. I will try to pin `matplotlib` to an older version until we fix this in conda-forge.

Comment (jorisvandenbossche) [inline recipe/meta.yaml]: yes, that sounds like a good idea!

What is actually the problem with matplotlib? (as the matplotlib builds in their feedstock seem fine)

Comment (ocefpaf) [inline recipe/meta.yaml]: `gdal` is compiled against `jpeg 9*`, but `matplotlib`'s `qt` is pinned against `jpeg 8*`. I believe that is the inconsistency that is here. What puzzles me is why this is affecting only python 3.5 on Windows.

**Comment by ocefpaf** [comment]
Argh... Something else is incompatible now... Debugging it...

**Pull request merged**